homeassistant/components/shelly/sensor.py

```diff
@@ -1458,6 +1453,127 @@ def __init__(
         state_class=SensorStateClass.MEASUREMENT,
         entity_class=RpcPresenceSensor,
     ),
+    "number_current_humidity": RpcSensorDescription(
+        key="number",
+        sub_key="value",
+        native_unit_of_measurement=PERCENTAGE,
+        suggested_display_precision=1,
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        role="current_humidity",
+    ),
+    "number_current_temperature": RpcSensorDescription(
```

Comment:
Same here:
`AABBCCDDEEFF-number:201-number` to `AABBCCDDEEFF-number:201-number_current_temperature`

Reply:
As discussed I will split these changes into another PR